x/evm/keeper/attest.go

```diff
@@ -77,33 +77,33 @@ func (k Keeper) attestRouter(ctx sdk.Context, q consensus.Queuer, msg consensust
 	ctx, writeCache := ctx.CacheContext()
 	defer func() {
 		if err != nil {
-			logger.WithError(err).Error("failed to attest. Skipping writeback.")
+			logger.With(err).Error("failed to attest. Skipping writeback.")
 			return
 		}
 		writeCache()
 	}()
 
 	consensusMsg, err := msg.ConsensusMsg(k.cdc)
 	if err != nil {
-		logger.WithError(err).Error("failed to cast to consensus message")
+		logger.With(err).Error("failed to cast to consensus message")
```

Comment:
Always use `WithError()` and `WithFields()`. 

Do not use `With()` for adding logging context.

This applies to all files in this PR. Please update accordingly.